plugins/modules/win_ntp.ps1

```diff
@@ -0,0 +1,183 @@
+#!powershell
+
+# Copyright: (c) 2020, Sakar Mehra <sakarmehra100@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#Requires -Module Ansible.ModuleUtils.Legacy
+
+$ErrorActionPreference = "Stop"
```

Comment:
Not needed anymore, this is always set by the PowerShell wrapper that executes modules.